src/current/_includes/v25.3/known-limitations/vector-limitations.md

```diff
@@ -1,8 +1,5 @@
 - {% include {{ page.version.version }}/sql/vector-batch-inserts.md %}
 - Creating a vector index through a backfill disables mutations ([`INSERT`]({% link {{ page.version.version }}/insert.md %}), [`UPSERT`]({% link {{ page.version.version }}/upsert.md %}), [`UPDATE`]({% link {{ page.version.version }}/update.md %}), [`DELETE`]({% link {{ page.version.version }}/delete.md %})) on the table. [#144443](https://github.com/cockroachdb/cockroach/issues/144443)
 - `IMPORT INTO` is not supported on tables with vector indexes. You can import the vectors first and create the index after import is complete. [#145227](https://github.com/cockroachdb/cockroach/issues/145227)
-- Only L2 distance (`<->`) searches are accelerated. [#144016](https://github.com/cockroachdb/cockroach/issues/144016)
```

Comment:
Looks like we added support for some but not all distance functions. It'd probably be good to mention the ones that we still don't support and reference https://github.com/cockroachdb/cockroach/issues/147839.

Reply:
Done!